Documentation/ceph-upgrade.md

```diff
@@ -268,7 +269,7 @@ Let's get started!
 > instructions to [migrate the Drive Group spec](#migrate-the-drive-group-spec) before performing
 > any of the upgrade steps below.
 
-## 1. Update common resources and CRDs
+### 1. Update common resources and CRDs
```

Comment:
In the rendered docs having a depth more than 3 in the title hierarchy makes the sections hard to distinguish. How about we keep the titles as they are?

Reply:
What if we instead make these bold? I keep finding it confusing to navigate the raw text when the hierarchy is arbitrary. I was also considering whether a TOC would make sense since the doc is rather long.

Reply:
Or we could change the stylesheet for the rendered docs to make the H3 and H4 headers stand out more.

Reply:
Right, perhaps if the H3 headers were bold it would really help. Let's keep it as you've changed it, then could look at the stylesheet separately.